gitlab/radiuss-jobs/lassen.yml

```diff
@@ -23,15 +23,38 @@ clang_14_0_5_gcc_8_3_1_cuda_11_7_0:
     MODULE_LIST: "cuda/11.7.0"
   extends: .job_on_lassen
 
+clang_16_0_6_ibm_gcc_11_2_1_cuda_11_8_0:
+  variables:
+    SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda %clang@=16.0.6.ibm.cuda.11.8.0.gcc.11.2.1 ^cuda@11.8.0+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS}"
+    MODULE_LIST: "cuda/11.8.0"
+  extends: .job_on_lassen
+
+clang_16_0_6_gcc_11_2_1_cuda_11_8_0:
+  variables:
+    SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda %clang@=16.0.6.cuda.11.8.0.gcc.11.2.1 ^cuda@11.8.0+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS}"
+    MODULE_LIST: "cuda/11.8.0"
+  extends: .job_on_lassen
+
 xl_2022_08_19_gcc_8_3_1_cuda_11_2_0:
   variables:
     SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda %xl@=16.1.1.12.gcc.8.3.1 ^cuda@11.2.0+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS}"
     MODULE_LIST: "cuda/11.2.0"
   extends: .job_on_lassen
 
+xl_2023_06_28_gcc_11_2_1_cuda_11_8_0:
+  variables:
+    SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda %xl@=16.1.1.14.cuda.11.8.0.gcc.11.2.1 ^cuda@11.8.0+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS}"
+    MODULE_LIST: "cuda/11.8.0"
+  extends: .job_on_lassen
+
 gcc_8_3_1_cuda_11_7_0:
```

Comment:
Do we need the CUDA 11.7 builds? I think most teams are on 11.2 or 11.8.

Reply:
Good catch. 11.2 is the default and what most teams are using. Some projects also test and work with 11.8. So let's go with 11.2 and 11.8 and nix 11.7

Reply:
OK, but 11.7 is often associated with gcc 11.2.1, we already have gcc 11.2.1 + cuda 11.8.
I think this particular spec was there to test with the newest cuda. I think we can remove it in favor of gcc 11.2.1 + cuda 11.8.

Reply:
works for me.

Reply:
Done.